### ISSUE TITLE
Include sample Storybook in template

### ISSUE THREAD

**Issue by jessicaaustin** [foxglove/fox#27]
The studio project makes use of Storybooks to help develop panels, which is great. It would be useful to have the Storybook infrastructure set up for the extension template created here as well, so we can have a place to start without setting that up ourselves.

In particular, this would be useful to have code to help set up a `PanelExtensionContext` for use in a storybook (or unit tests for that matter). For example, I'm looking at the Map and Teleop panels, and they both make use of `studio-base/components/PanelExtensionAdapter` and `studio-base/stories/PanelSetup`. I've spent some time trying to either import those or build something similar so I can get my custom panel loaded in a storybook, and keep running into issues. 

(For context, I'm a React beginner and in general not a frontend expert, so I may be missing something obvious here)

**Comment by defunctzombie** [comment]
Thanks for the feedback.



> (For context, I'm a React beginner and in general not a frontend expert, so I may be missing something obvious here)

React isn't required for making a panel extension. This was a deliberate decision so that folks not as familiar with React and its nuances - or have prior non-react frontend experience - could cobble together something using techniques they might know. We'll see how this plays out over time tho.

We can add a mode to `fox` that does produce a react ready example with storybook to help folks get going that do want to use React. Or we could make a few example projects. @jhurliman what do you think?

**Comment by jessicaaustin** [comment]
> React isn't required for making a panel extension.

Yep, understood, but the current template uses it (`ExamplePanel.tsx`), and at least right now everything in Studio uses React as well, so I figured it would be easier to start there since there are more examples to reference. The turtlesim extension is a non-react example, but that's incomplete.

So personally I would find it helpful to have a "react mode" like you suggest.

Thanks!
